Tools/src/FormatCheckTool/FormatCheckTool.cs

```diff
@@ -0,0 +1,234 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Microsoft.Recognizers.Text;
+using Microsoft.Recognizers.Text.Choice;
+using Microsoft.Recognizers.Text.DateTime;
+using Microsoft.Recognizers.Text.Number;
+using Microsoft.Recognizers.Text.NumberWithUnit;
+using Microsoft.Recognizers.Text.Sequence;
+using Newtonsoft.Json;
+using Microsoft.Recognizers.Text.DataDrivenTests;
+
+namespace FormatCheckTool
+{
+    class FormatCheckTool
```

Comment:
ValidationTool may be a better name as it's not only going to check for format issues.